### PR TITLE
adds it assistance page

### PR DESCRIPTION
**Pull request by rbenson09**
need to check the format before I add more content. the menu item was also added to the config.yml

### CODE REVIEW

**Comment by FelicianoTech** [comment]
@rbenson09 How do you want me to go about this review? Would you rather I point out problems with suggestions? Or would you rather I fix the problems?

**Comment by rbenson09** [comment]
@felicianotech either way is fine. i don’t mind fixing it myself after your review.

File: src/content/all/it-assistance.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+**Note: This is for Windows systems only**
+
+ 1. Visit the [DoD Cyber Exchange](https://public.cyber.mil/pki-pke/pkipke-document-library/?_dl_facet_pkipke_type=tools) website (created by DISA) to download InstallRoot. 
+ 2. Scroll to the bottom of the page until you find "InstallRoot 5.X: NIPR Windows Installer". Most systems will be 64-bit.
+ 3. Click on the installer, save to your local computer, and run the program.
+ 4. Click "Next" all the way through and then click "Install".
+ 5. Once InstallRoot is installed, run the program. Click "Install Certificates" and then click "Ok".
+ 6. Restart Internet Explorer and try to open a DoD, CAC-enabled website.
+
+{{<figure src="/assets/img/content/install-root-install-certificates.jpg"class="align center small">}}
```
Comment (FelicianoTech) [inline src/content/all/it-assistance.md:22]: There needs to be a space after `certificates.jpg"`.

Also, `align center` should be `aligncenter`.

**Review by FelicianoTech — CHANGES_REQUESTED** [review]
I made some comments.

Also, the three images you're trying to use weren't included in this PR/commit. This is why CircleCI is failing.

**Review by FelicianoTech — APPROVED** [review]
Looks good to me.

**Pull request merged**